packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandStringOptionMinMaxLengthMixin.ts

```diff
@@ -0,0 +1,18 @@
+export abstract class ApplicationCommandStringOptionMinMaxLengthMixin {
```

Comment:
Can't this just be part of the `SlashCommandStringOption`, rather than a mixin since it's only applicable to string options?